js/adapt-contrib-bookmarking.js

```diff
@@ -109,6 +165,19 @@ class Bookmarking extends Backbone.Controller {
     this.stopListening(Adapt, 'bookmarking:cancel');
   }
 
+  navigateToFurthest() {
+    const furthestModel = this.checkFurthestIncompleteModel();
+    const furthestId = furthestModel.attributes._id;
```

Comment:
```suggestion
    const furthestModel = this.furthestIncompleteModel;
    const furthestId = furthestModel.get('_id');
```
`model.attributes` is a private property in backbone and shouldn't be used directly